components/Card/card.css

```diff
@@ -0,0 +1,229 @@
+@import url('https://fonts.googleapis.com/css2?family=Amatic+SC:wght@700&family=Andika&family=Architects+Daughter&family=Balsamiq+Sans&family=Dancing+Script:wght@600&family=Dongle&family=Lato&family=Open+Sans:wght@300&family=Roboto:wght@400;500&display=swap');
```

Comment:
why you are repeating this importing font and defining color.
do it in one separate file and import that file